doc-files/README.tmpl.md

```diff
@@ -0,0 +1,133 @@
+# h3-js
```

Comment:
Note that this is unchanged from the old README, other than the addition of the placeholder for docs.

Reply:
We might want to add a comment to README.md (via this template) that the file is generated.

Reply:
Hm. This was a little tricky to do without putting the "generated" comment in the template, which I think would be confusing (as the template itself is not generated), but I got it working with `sed`. I'm not sure this is worthwhile, but it might save someone a little time down the road.

Reply:
It looks like GitHub doesn't respect HTML comments (they'd be visible in the rendered README), so I'm using the alternative Markdown comment syntax proposed here: https://stackoverflow.com/a/32190021/380487
```
[NOTE] # (This file is generated via build-docs, do not edit!)
```